extra/patterns/diamond.js

```diff
@@ -0,0 +1,42 @@
+let n,space=1;
```

Comment:
Add comments to your code showing the expected output and a brief explanation of your function in each necessary line

Reply:
Done